src/app/home/directives/home-story.component.ts

```diff
@@ -88,10 +88,10 @@ export class HomeStoryComponent implements OnInit {
       this.statusText = 'Draft';
     } else if (this.story.changed()) {
       this.statusClass = 'status unsaved';
-      this.statusText = 'Edited';
+      this.statusText = 'Unsaved Changes';
     } else if (!this.story.publishedAt) {
       this.statusClass = 'status unpublished';
-      this.statusText = 'Unpublished';
+      this.statusText = 'Private';
```

Comment:
Yesss!  (I'm a fan of calling things public/private instead of published/unpublished).  Eventually we'll also want to move the "make private again" button somewhere less prominent.  But that's another UX problem for another day.